rules-reviewed/technology-usage/configuration-management.windup.xml

```diff
@@ -27,5 +27,69 @@
                 <technology-tag level="INFORMATIONAL">Spring Cloud Config</technology-tag>
             </perform>
         </rule>
+        <rule id="configuration-management-0200">
+            <when>
+                <file filename="application-{profile}.{extension}"   />
+            </when>
+            <perform>
+                <classification title="Spring application properties detected" category-id="information" effort="0">
+                    <description>Spring application properties detected</description>
+                </classification>
+                <technology-tag level="INFORMATIONAL">Spring Properties</technology-tag>
+            </perform>
+            <where param="profile">
+                <matches pattern=".+" />
+            </where>
+            <where param="extension">
+                <matches pattern="(properties|yml|yaml)" />
+            </where>
+        </rule>
```

Comment:
This rule is to find files named in the format 'application-xxxx.properties' (or .yml or .yaml). Do we want it to also find files named just 'application.properties' , as in the rule below it? If so, the profile pattern needs to be the same as that in rule ..-0300